src/chi/utilities/display.scss

```diff
@@ -18,3 +18,27 @@
     }
   }
 }
+
+@media screen {
+  .-d-print--only {
+    display: none;
+  }
+
+  .-d-screen--only {
+    display: block;
+  }
+}
+
+@media print {
+  .-d-print--only {
+    display: block;
+
+    .chi-data-table__print-footer {
+      padding: 1rem;
+    }
```

Comment:
```suggestion
```

Reply:
As we agreed, the `-d-print--only`, `-d-screen--only` are generic display classes that should not have attached any data table-specific properties. Please define the class in `data-table.scss`.